### ISSUE TITLE
ElementMesh generated misses PointElements

### ISSUE THREAD

**Issue by ruebenko** [c3m-labs/GmshLink#2]
When you run this code:

```
r = RegionUnion[Cylinder[], Cuboid[]];
mesh = ToElementMesh[r, "BoundaryMeshGenerator" -> None, 
  "ElementMeshGenerator" -> {GmshGenerator}, 
  MaxCellMeasure -> Infinity, "MeshOrder" -> 1]
(*mesh["Wireframe"]*)
mesh["PointElements"]
{PointElement[{{1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}}, {0, 0, 0, 0, 
   0, 0, 0, 0}]}
```

you see that the PointElement only have 8 incidents. Looking at the wireframe this does not seem correct. Most likely there should be as many point elements as there are unique boundary element incidents.

**Comment by PintarM** [comment]
Thank you for reporting this issue. I apologize for the late reply, but somehow I missed this message.

Limited number of `"PointElements"` is a result of faithfully importing `.msh` file, which contains mesh generated by GMSH. You can check this by using the option `..."ElementMeshGenerator" -> {GmshGenerator, "DeleteFiles" -> False},...` and manually inspecting intermediate file ("GmshLink.msh").

I understand that by default `ElementMesh` has as many `PointElement` as there are nodes on the boundary, but I think `.msh` file should be imported as it is. Users can then get outer nodes with something like:

```mathematica
outerNodes[mesh_ElementMesh] := With[{
   vbc = mesh["VertexBoundaryConnectivity"]
   },
   Union@First@Transpose[(vbc.Transpose[vbc])["NonzeroPositions"]]
]
```
Do you agree?

**Comment by ruebenko** [comment]
Possibly this is a bug in GMSH. I do not understand what those 
points that gmsh produces are. Maybe it's better then to not import the 
point elements if they are not correct or incomplete. In that case 
ElementMesh will figure out by itself what the points on the boundary are.

My feeling is that the ElementMesh generated should be correct.

**Comment by PintarM** [comment]
Aha, good idea, I get it.
In my opinion the main goal of this package is to produce quality 3D mesh via GMSH and therefore elements of lower dimensions are less important. I will just import 3D `"MeshElements"` and let `ElementMesh` to take care of the rest.

**Comment by ruebenko** [comment]
Yes, that is probably a good idea. The only case where this may not be 
sufficient is when GMSH has an internal boundary. But I am not even sure 
GMSH can generate meshes with internal boundaries. If such a case ever 
comes up then the code could be adjusted.

**Comment by PintarM** [comment]
Closed with commit `ea608d90e593e29f01df9850b0fa90c84cd3a85f`.
